pylabrobot/io/serial.py

```diff
@@ -183,6 +183,22 @@ def _open_serial() -> serial.Serial:
         self._executor = None
       raise e
 
+    assert self._ser is not None
+
+    # --- FIX: Prevent FTDI reset on open/close (critical for Inheco on Raspberry Pi) ---
+    try:
+      # Some pyserial versions require direct attribute access:
+      self._ser.dtr = False
+      self._ser.rts = False
+
+      # Others only respect the explicit setter:
+      self._ser.setDTR(False)
+      self._ser.setRTS(False)
```

Comment:
do you know which versions these are?

Reply:
ok but where did you notice you have to use one or the other? is one even supported on platforms where the other isn't?